source/jormungandr/jormungandr/street_network/here.py

```diff
@@ -251,13 +251,27 @@ def _read_response(response, origin, destination, mode, fallback_extremity, requ
             fm.car_no_park.name: response_pb2.Car,
         }
         section.street_network.mode = map_mode[mode]
+
+        # handle maneuvers to fill Path field
+        def _convert_direction(direction):
```

Comment:
I'm not sure we should show this information which is potentially false...

Reply:
what's is instruction when a roundabout is involved?

Reply:
This is Here isntruction exemple for roundabout
```
"Take the <span class="exit" ="">1st exit</span> from Place François Mitterrand roundabout onto <span class="next-street">Boulevard Périphérique</span> toward <span class="sign"><span lang="fr">Nancy</span>/<span lang="fr">Aéroport Orly</span>/<span lang="fr">Lyon</span>/<span lang="fr">Porte de Vincennes</span>/<span lang="fr">Périphérique Intérieur</span></span>. <span class="distance-description">Go for <span class="length">2.1 km</span>.</span>"
```
To handle the `direction` field, the rules is simplified. **Here API** offers a `direction` field the can be :

- right
- left
- forward
- bearRight
- etc...

We only convert right to 90 and left to -90 and the others equals to 0 to be backward compatible.